tests/unit/utils/event_test.py

```diff
@@ -294,6 +294,19 @@ def test_event_many_backlog(self):
                 evt = me.get_event(tag='testevents')
                 self.assertGotEvent(evt, {'data': '{0}'.format(i)}, 'Event {0}'.format(i))
 
+    # Test the fire_master function. As it wraps the underlying fire_event,
+    # we don't need to perform extensive testing.
+    def test_send_master_event(self):
+        """"Tests that sending an event through fire_master generates expected event"""
+        with eventpublisher_process():
+            me = event.SaltEvent("minion", SOCK_DIR)
+            data = {"data": "foo1"}
+            me.fire_master(data, "test_master")
+
+            evt = me.get_event(tag="fire_master")
+            self.assertGotEvent(evt, {"data": data, "tag": "test_master", "events": None, "pretag": None})
+
+
```

Comment:
@aurynn Did this test pass for you after you wrote it? It's not passing for me and seems to be failing in Jenkins as well. I'm not sure we can use this technique for checking for events fired across the networks bus like this. We might need to find a different approach here.